dev-requirements.in

```diff
@@ -35,11 +35,13 @@ torch<=2.0.0; python_version>='3.11' or platform_system!='Windows'
 # Once a solution is found, this should be updated to support Windows as well.
 python-magic; (platform_system=='Darwin' or platform_system=='Linux')
 
-pillow
-scikit-learn
 types-protobuf
 types-croniter
 types-mock
 autoflake
+
+pillow
```

Comment:
Do you think this would be too annoying for local development? We'll remember to install pandas + pyarrow to enable some for the test suite.

I am okay with it. For others maybe a comment:

```suggestion

# Optional runtime dependnecies: Uncomment to enable tests on pandas
# pandas

pillow
```

Reply:
Added pandas to dev-requirement, and uninstall it in the GA workflow instead.